std_msgs/Header header
PlanarFeature[] features
